sass/atoms/_link-button.scss

```diff
@@ -0,0 +1,36 @@
+.button {
+    position: relative;
+    display: block;
+    overflow: hidden;
+    width: auto;
+    cursor: pointer;
+    letter-spacing: 0.5px;
+    font-size: 1.5rem;
+    margin: 40px auto;
+    padding: 15px;
+    text-align: center;
+
+    &:hover {
+        color: #fdb924;
+    }
+}
+
+/*important*/
+
+a.button.tertiary:hover:before {
+    width: 10%;
+}
+
+a.button.tertiary:before {
+    position: absolute;
+    content: '';
+    height: 2px;
+    bottom: 0;
+    margin: 0 auto;
+    left: 0;
+    right: 0;
+    width: 5%;
+    background: #fdb924;
+    transition: width .1s ease-in-out;
+    color: inherit;
+}
```

Comment:
nest these selectors inside of .button
There's no need to write it as a.button, .button is enough.